microsoft-edge/webview2/concepts/working-with-local-content.md

```diff
@@ -218,14 +298,48 @@ webView->NavigateToString(htmlString);
 
 
 <!-- ====================================================================== -->
-## Virtual host name mapping
+## Loading local content by using virtual host name mapping
 
 Another way to load local content in a WebView2 control is to use virtual host name mapping.  This involves mapping a local domain name to a local folder, so that when the WebView2 control attempts to load a resource for that domain, it will load the content from the specified local folder location instead. The origin of the document will also be the virtual host name. 
 
 Due to a current limitation, media files that are accessed using a virtual host name can be slow to load.<!-- when limitation is fixed, remove sentence -->
 
 This approach lets you specify the cross-origin access, by using the `CoreWebView2HostResourceAccessKind` enum.
 
+
+<!-- ------------------------------ -->
+#### Considerations for loading local content by using virtual host name mapping
+
+
+###### Origin-based DOM APIs
+
+Local content loaded via virtual host name mapping has an http or https URL and a corresponding origin. So APIs that require an origin such as `localStorage` or `indexedDB` will work, and the usual ["same-origin" rules](https://developer.mozilla.org/en-US/docs/Web/Security/Same-origin_policy) apply.
+
+
+###### DOM APIs requiring secure context
+
+Some newer browser features are limited to https URLs. Using virtual host name mapping essentially provides https URLs for your local content. This means APIs such as webcam APIs, geolocation APIs, and notification APIs are all available. For more information, see [Secure contexts](https://developer.mozilla.org/en-US/docs/Web/Security/Secure_Contexts).
+
+
+###### Dynamic content
+
+When loading local content via a virtual host name mapping, you are mapping a virtual host name to a local folder of static files. Just like file URLs, there is no opportunity to dynamically modify the local content while it is loaded by the virtual host name.
+
+
+###### Additional web resources
+
+Local content loaded via virtual host names have http and https URLs. These URLs support relative URL resolution, and so an HTML document served via virtual host names can have references such as CSS, script, and image references that are also served via virtual host names.
```

Comment:
```suggestion
Local content that's loaded via virtual host names has http or https URLs. These URLs support relative URL resolution, and so an HTML document that's served via virtual host names can have references such as CSS, script, and image references that are also served via virtual host names.
```
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/t/that-vs-which -- "Include that even if the sentence is clear without it. It helps to clarify the sentence structure."
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/u/using-by-using-with -- "In instructions or technical content, use by using or that use to eliminate ambiguity."